docs/rst/manual/developer_documentation/index.rst

```diff
@@ -7,3 +7,6 @@
   concepts
   forward_model
   communication_protocol
+
+.. automodule:: ert3.workspace._workspace
+  :members:
```

Comment:
Could we have these modules under a designated heading (called modules or something better)? I think it will be a bit cluttered if we are to have all of these as top level headings.